src/lib/components/RadarChart.js

```diff
@@ -16,12 +16,18 @@ const defaultOptions = {
   axes: true, // show axes?
   scales: 3, // show scale circles?
   captions: true, // show captions?
+  dots: false, // show dots?
   zoomDistance: 1.2, // where on the axes are the captions?
   smoothing: noSmoothing, // shape smoothing function
   captionMargin: 10,
   axisProps: () => ({ className: 'axis' }),
   scaleProps: () => ({ className: 'scale', fill: 'none' }),
   shapeProps: () => ({ className: 'shape' }),
+  dotProps: () => ({
+    className: 'dot',
+    mouseEnter: (dot) => { console.log(dot) },
```

Comment:
please remove console.log

Reply:
I can see why you would want to remove `console.log`s from the code.
The only smart thing to do then would be omitting the functions completely from the default options, because the `dot` value would be unreachable if the user doesn't provide a function. So would you like me to remove it and only mention the functions it in the README?

Reply:
Sure, you can mention it in the README or use it in the sample app.